src/sentry/api/serializers/models/group.py

```diff
@@ -225,6 +226,17 @@ def get_attrs(self, item_list, user):
                     safe_execute(plugin.get_annotations, group=item, _with_transaction=False) or ()
                 )
 
+            for integration in Integration.objects.filter(
+                    organizations=item.project.organization_id):
+                if not (integration.has_feature(IntegrationFeatures.ISSUE_BASIC) or integration.has_feature(
+                        IntegrationFeatures.ISSUE_SYNC)):
+                    continue
```

Comment:
Could this be `if hasattr('get_annotations', integration)`? Then implementing the method gets you the annotation and there are fewer stars to align when building new integrations.

Reply:
Hmm.. @markstory I think I support using the integration feature flags in this case, given its how we've handled other integration features. I like the idea from the standpoint of being able to search Issue_basic or Issue_sync to find all the places we've implemented those specific features as well from a practical maintainer's standpoint.

Reply:
Ok.